docs/concepts/client/index.md

```diff
@@ -150,15 +150,15 @@ Client(host='https://my.awesome.flow:1234', port=4321)
 
 ````{admonition} Caution
 :class: caution
-In case you instanciate a `Client` object using the `grpc` protocol, keep in mind that `grpc` clients cannot be used in 
-a multi-threaded environment (check [this gRPC issue](https://github.com/grpc/grpc/issues/25364) for reference).
+
+According to [this gRPC issue](https://github.com/grpc/grpc/issues/25364), `grpc` clients cannot be used in 
+a multi-threaded environment. So we apply `RLock` as a workaround. 
```

Comment:
do not call it a workaround

Reply:
Other than what @JoanFM said, I don't see any problem here. Fix the "workaround" thing and ask me for a re-review :wink: